Automattic/wp-calypso
# Stats: Modernising Subscribers on the Insights page

Related to #72134

## Proposed Changes

* modernising the `Subscribers` module behind a feature flag (Stats Insights page)
* extending Stats card with horizontal bars to display text values instead of numbers and avoid showing horizontal bars
* apply flex box wrapping to the header (only the one having a toggle) because this component is very narrow as one or 3 modules in one row
* converting an option to display an avatar from a specific module to a property 
* replacing moment.js time calculation

## Testing Instructions



* go to the live branch and the `Insights` page
* verify that the current `Subscribers` module renders correctly and select switching data between wp.com and email works correctly
* apply feature flag to the URL `?flags=stats/insights-page-grid`
* verify that the new toggle in the header switches between data sets

| Before | After |
| --- | --- |
| ![SCR-20230216-eif](https://user-images.githubusercontent.com/112354940/219171777-a0887b07-feaa-40d2-b2ba-a0d9821d080b.png) | ![SCR-20230216-ehc](https://user-images.githubusercontent.com/112354940/219171854-56e9409a-518b-4d9b-a33f-f76121fa842a.png) |

## Pre-merge Checklist
